### ISSUE TITLE
errors when working with spelling and grammar correction in word

### ISSUE THREAD

**Issue by fernando-jose-silva** [nvaccess/nvda#8619]
### Steps to reproduce:
open the word and enter a text with several grammatical and hortographic errors.
### Actual behavior:
press ctrl + home to go to the beginning of the text.
Press alt + f7
Word selects the word with the error.
Nvda does not say anything, however it is possible to see the selected text by pressing nvda + shift + s.
my suggestion would be for nvda to pronounce the selected text automatically.
Now go to the beginning of the text again and press f7
Note that nvda does not speak about the error found in the text, it is possible to see the suggested correction by walking with tab.
My suggestion would be for nvda to pronounce the text with errors automatically.
### Expected behavior:
When word selects the wrong word or phrase, nvda should automatically read the text with errors.
When pressing alt + f7, nvda should read the text pointed to with errors by word.
#### NVDA Installed/portable/running from source:
instaled
#### NVDA version:
alfa 15781
#### Windows version:
10 17134.191
#### Name and version of other software in use when reproducing the issue:
word 16.0.10325.2082
#### Does the issue still occur after restarting your PC?
yes
#### Have you tried any other versions of NVDA?
no

**Comment by Brian1Gaff** [comment]
It does this on older versions of Word, though if using a development snap 
it usually has one or more beeps for non fatal errors as well.
 Brian

bglists@blueyonder.co.uk
Sent via blueyonder.
Please address personal E-mail to:-
briang1@blueyonder.co.uk, putting 'Brian Gaff'
in the display name field.
----- Original Message -----

**Comment by leonardder** [comment]
Is this Word with or without UIA enabled?

**Comment by fernando-jose-silva** [comment]
I do not know, I did not change the settings for this in nvda

 

De: Leonard de Ruijter <notifications@github.com> 
Enviada em: sábado, 11 de agosto de 2018 14:30
Para: nvaccess/nvda <nvda@noreply.github.com>
Cc: fernando-jose-silva <fernandoj.silva21@gmail.com>; Author <author@noreply.github.com>
Assunto: Re: [nvaccess/nvda] errors when working with spelling and grammar correction in word (#8619)

 

Is this Word with or without UIA enabled?

**Comment by Adriani90** [comment]
@fernando-jose-silva could you please be more clear on the expected behavior here? When I press alt+f7 the first proposal for the correct word is reported which is expected in my opinion. You already hear "spelling error" while typing if you turn that setting on in NVDA. So you know that the word is spelled wrong and you get the correct version when you press alt+f7 which is reported.

**Comment by fernando-jose-silva** [comment]
let me disagree, if I have been typing a text for several months, which is very common in my work, when pressing alt + f7 I do not know which word world highlighted as an error.  I do not correct text as I type, I correct after several months when I have finished the entire job.  yes i hope nvda announces the wrong word when i press alt + f7

Enviado do meu iPhone

**Comment by Adriani90** [comment]
@fernando-jose-silva it seems this shortcut does not work in MS Word 365. are you still having this issue in MS Word 2016?

**Comment by fernando-jose-silva** [comment]
yes, in my opinion this request is still valid, this shortcut does not work if you happen to press the F7 key alone just once.
